src/utils/generator/ReplicaFiller.cpp

```diff
@@ -64,19 +64,19 @@ class ParticleContainerToBasisWrapper : public ParticleContainer {
 	unsigned long getNumberOfParticles() override { return _basis.numMolecules(); }
 
 	double getBoundingBoxMin(int dimension) const override {
-		double min[3];
-		_object->getBboxMin(min);
+		double min[3] = {std::numeric_limits<double>::min(),std::numeric_limits<double>::min(),std::numeric_limits<double>::min()};
+	//	_object->getBboxMin(min);
 		return min[dimension];
 	}
 
 	double getBoundingBoxMax(int dimension) const override {
-		double max[3];
-		_object->getBboxMax(max);
+		double max[3] = {std::numeric_limits<double>::max(),std::numeric_limits<double>::max(),std::numeric_limits<double>::max()};
+	//	_object->getBboxMax(max);
 		return max[dimension];
 	}
 
 	bool isInBoundingBox(double r[3]) const override {
-		_object->isInside(r);
+		true;
```

Comment:
leave the old code commented out, but in, so that the conceptual idea is not lost, as above.